victorops-twilio.js

```diff
@@ -910,159 +914,158 @@ function isHuman (twiml, context, event, payload) {
         )
       );
     }
-
     resolve(twiml);
   });
 }
 
 // Records caller's message and transcribes it
 function leaveAMessage (twiml, context, event, payload) {
   log('leaveAMessage', event);
-  return new Promise((resolve, reject) => {
     const {messages, NO_VOICEMAIL} = context;
-    const {DialCallStatus} = event;
+    const {DialCallStatus, DialBridged} = event;
     const {callerId, detailedLog, goToVM, teamsArray, sayGoodbye, voice, realCallerId} = payload;
-
+    
     // Caller was connected to on-call person and call completed
-    if (DialCallStatus === 'completed') {
-      twiml.say(
-        {voice},
-        `${messages.otherPartyDisconnect} ${messages.goodbye}`
-      );
-    // If caller does not hang up after leaving message,
-    // this message will play and then end the call
-    } else if (sayGoodbye === true) {
-      twiml.say(
-        {voice},
-        `${messages.attemptTranscription} ${messages.goodbye}`
-      );
+    if (DialCallStatus == 'completed' && DialBridged == 'true') {
```

Comment:
if (DialCallStatus === 'completed' && DialBridged == 'true') {

Reply:
Using === prevents recovery alerts when the callee hangs up.

Reply:
Whatever gets the result we want :)